integration_tests/pom.xml

```diff
@@ -81,6 +81,10 @@
             <groupId>org.rogach</groupId>
             <artifactId>scallop_${scala.binary.version}</artifactId>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
```

Comment:
This is not a light dependency.  Do we really need it?  If we're running under the Spark classpath (or can do so simply), that would be far preferable to pulling in our own here, IMO.